public/views/login.html

```diff
@@ -1,6 +1,28 @@
-<ion-view title="login">
-  <ion-content class="padding">
-    <h1> This is the login page</h1>
-    <a ui-sref="signup"></a>
-  <ion-content>
-<ion-view>
\ No newline at end of file
+<ion-view>
+  <ion-content class="login">
+
+  <div>
+    <h1 class="login-title">Login</h1>
+    <form name="loginForm">
+        <input name="username" type="text" ng-model="user.username" placeholder="Username" ng-minlength="6" required>
+        <input name="password" type="text" ng-model="user.password" placeholder="Password" ng-minlength="6" ng-patter="passwordValidation" required>
+    </form>
+
+      <div ng-show='loginForm.username.$error.required || loginForm.password.$error.required'>
+        <h3 class="login-text">Username and Password Required</h3>
+      </div>
+      <div ng-show='loginForm.username.$error.minlength || loginForm.password.$error.minlength'>
+        <h3 class="login-text">Username and Password Must Be At Least 6 Characters</h3>
+      </div>
+      <div ng-show='loginForm.password.$error.pattern'>
+        <h3 class="login-text">Password Must Contain Both A Number and A Capital Letter</h3>
+      </div>
+        
+      <a href='#/signup'>Need to signup for an account? <strong class="strong-text">Signup</strong> ...</a>
```

Comment:
We want to use `ui-sref` instead of `href` everywhere so that if we decide to change the paths later (like remove the `#` from all the links, or change the path for this one to `/register`) we don't have to change any of the code, since `ui-sref` refers to the state name, regardless of what the path is.

So this should be `<a ui-sref="signup">`.